extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/hll/sql/HllSketchSqlAggregatorTest.java

```diff
@@ -852,7 +855,11 @@ public void testEmptyTimeseriesResults()
         ImmutableList.of(Druids.newTimeseriesQueryBuilder()
                                .dataSource(CalciteTests.DATASOURCE1)
                                .intervals(querySegmentSpec(Filtration.eternity()))
-                               .filters(bound("dim2", "0", "0", false, false, null, StringComparators.NUMERIC))
+                               .filters(
+                                   NullHandling.replaceWithDefault()
+                                   ? numericSelector("dim2", "0", null)
+                                   : equality("dim2", 0L, ColumnType.LONG)
+                               )
```

Comment:
I had this comment above and I think the difference might be the `numericSelector`?  If so, couldn't the `equality` look at the type parameter to figure out if it should create a "normal" selector or a numeric one?